src/odb/src/db/dbTechLayer.cpp

```diff
@@ -733,6 +746,7 @@ dbIStream& operator>>(dbIStream& stream, _dbTechLayer& obj)
   stream >> *obj.width_table_rules_tbl_;
   stream >> *obj.min_cuts_rules_tbl_;
   stream >> *obj.area_rules_tbl_;
+  stream >> *obj.pitch_rules_tbl_;
```

Comment:
I don't understand, what version check?

Reply:
Here are the instructions: https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/odb/doc/OpenDB-AddFieldsInDbObjects.md

Reply:
If some has an old db without the new field we still want to be able to load it.  Look at how db_schema_wrongway_width is setup to see how we handled it for another addition.

Reply:
I am afraid that this part is handled by the db code generator and editing it would require editing the code generator itself to handle the version check. When we started the code generator, there was no db version to handle.

Reply:
That's unfortunate, the right solution would be to fix the code generator so we can have backwards compatibility

Reply:
@osamahammad21 we could add a schema_check field to the json and a generate a corresponding check here.  It doesn't seem like a big change to the generator.  What do you think?